.ci/install_kata_kernel.sh

```diff
@@ -174,23 +68,16 @@ main() {
 
 	case "${kernel_type}" in
 		experimental)
-			install_kernel "${kernel_type}" $(get_version "assets.kernel-experimental.tag")
+			build_and_install_kernel "kernel-experimental"
 			;;
 		vanilla)
-			install_kernel "${kernel_type}" $(get_version "assets.kernel.version")
-			;;
-		tdx)
-			install_kernel "${kernel_type}" $(get_version "assets.kernel.tdx.tag")
-			;;
-		sev)
-			install_kernel "${kernel_type}" $(get_version "assets.kernel.sev.tag")
+			build_and_install_kernel "kernel"
 			;;
 		dragonball)
-			install_kernel "${kernel_type}" $(get_version "assets.kernel-dragonball-experimental.version")
+			build_and_install_kernel "kernel-dragonball-experimental"
 			;;
 		*)
-			info "kernel type '${kernel_type}' not supported"
-			usage 1
+			die_unsupported_kernel_type "$kernel_type"
```

Comment:
I don't see where this call is defined... Am I missing something?
All "die()" calls are in the form "die [message]", is it what you intended?